elide-core/src/main/java/com/yahoo/elide/core/EntityDictionary.java

```diff
@@ -169,10 +169,12 @@ public static Method findMethod(Class<?> entityClass, String name, Class<?>... p
     }
 
     protected EntityBinding getEntityBinding(Class<?> entityClass) {
-        if (isMappedInterface(entityClass)) {
-            return EMPTY_BINDING;
-        }
-        return entityBindings.getOrDefault(lookupEntityClass(entityClass), EMPTY_BINDING);
+        return entityBindings.computeIfAbsent(entityClass, cls -> {
+            if (isMappedInterface(cls)) {
```

Comment:
This looks like it might be a merge error.  I would trust the AggregationDataStore branch over your own here.